lib/electric_slide/call_queue.rb

```diff
@@ -219,6 +219,7 @@ def conditionally_return_agent(agent, return_method = @agent_return_method)
         return_agent agent
       else
         logger.debug "Not returning agent #{agent.inspect} to the queue"
+        return_agent agent, :after_call
```

Comment:
That "mode" is https://github.com/adhearsion/electric_slide/blob/develop/lib/electric_slide/call_queue.rb#L25-L28